store/tikv/coprocessor.go

```diff
@@ -661,10 +686,16 @@ func (it *copIterator) Next(ctx context.Context) (kv.ResultSubset, error) {
 			if ok {
 				break
 			}
+			// record finished taskID
+			finishedTaskID := it.tasks[it.tasksStatus.curr].id
```

Comment:
It seems that `it.tasks[it.tasksStatus.curr].id` must equal to `it.tasksStatus.curr`?

Reply:
yes